insights/client/utilities.py

```diff
@@ -445,3 +447,41 @@ def os_release_info():
         except Exception as e:
             logger.warning("Failed to detect OS version: %s", e)
     return (os_family, os_release)
+
+
+def largest_spec_in_archive(archive_file):
+    logger.info("Checking for large files...")
+    tar_file = tarfile.open(archive_file, 'r')
+    largest_fsize = 0
+    # get the name of the archive
+    for elems in archive_file.split("/"):
+        if "tar" in elems:
+            name = elems.split(".", 1)
+    name[0]
```

Comment:
I think this is a leftover or something else. But I guess you can get rid of this line.

Reply:
Yes, it is a leftover